### PR TITLE
Back end documentation

### PR DESCRIPTION
**Pull request by Kyhl**
Comments on the back-end

### CODE REVIEW

File: PolloPollo.Repository/DonorRepository.cs [hunk 1]
```diff
@@ -30,22 +30,23 @@ public DonorRepository(IOptions<SecurityConfig> config, IPolloPolloContext conte
         }
 
 
-
+        /// <summary>
+        /// Update the information of a donor already in the database.
+        /// </summary>
```
Comment (thomasdal) [inline PolloPollo.Repository/DonorRepository.cs:35]: Does this update the information?

File: README.md [hunk 2]
```diff
@@ -90,7 +76,7 @@ dotnet ef migrations script -i --project ../PolloPollo.Entities -o <FILE>
         },
     }
     ```
-    2. ```> dotnet publish``` - This command builds the code and creates the directory with the builded dll's at the path: PolloPollo.Web/bin/Debug/netcoreapp2.2/publish
+    2. ```dotnet publish``` - This command builds the code and creates the directory with the builded dll's at the path: PolloPollo.Web/bin/Debug/netcoreapp2.2/publish
```
Comment (thomasdal) [inline README.md]: Not correct path

**Review by thomasdal — COMMENTED** [review]
Check comments

File: README.md (hunk 2 above)
Comment (joglr) [reply README.md:79]: Maybe we should use `dotnet publish -c Release` ?

Comment (nimrossum) [reply README.md]: Should be fixed now.

**Review by nimrossum — APPROVED** [review]
Great job!

**Pull request merged**